test/test_helper.rb

```diff
@@ -1,10 +1,10 @@
 require 'simplecov'
 
-SimpleCov.start 'rails'
+# SimpleCov.start 'rails'
 # MiniTest changes mean that our coverage suddenly dropped, since more controllers are being tested.
 # We dropped the value to 77 to be able to get pull requests pulled in. Needs to be brought back up as coverage goes back up.
-SimpleCov.minimum_coverage 77
-Mongo::Logger.logger.level = Logger::WARN
+# SimpleCov.minimum_coverage 77
```

Comment:
Did we get rid of Simplecov? If so, we should figure out another way to do code coverage, because we probably don't want to lose that feature

Reply:
I'm not actually sure when this happened, I would really suggest we consider https://codecov.io/ since it is free for open source and provides us verification that coverage has not dropped on tests.

Reply:
Also since the tests are run on multiple different workers now it is not possible to merge results natively, we would need codecov for that.